test/test_prototype_builtin_datasets.py

```diff
@@ -180,3 +180,20 @@ def test_label_matches_path(self, test_home, dataset_mock, config):
         for sample in dataset:
             label_from_path = int(Path(sample["path"]).parent.name)
             assert sample["label"] == label_from_path
+
+
+@parametrize_dataset_mocks(DATASET_MOCKS["usps"])
+class TestUSPS:
+    def test_label_matches_path(self, test_home, dataset_mock, config):
```

Comment:
@pmeier what is the goal of this test?
Is `test_label_matches_path` an accurate name? I don't see an obvious path check

Reply:
My bad, I missed the incorrect name. Should be something like

```suggestion
    def test_sample_content(self, test_home, dataset_mock, config):
```

The test checks what #5246 tried to add in a general way. Although USPS would be the only test right now that checks this, IMO it doesn't hurt. If we go through with #5246, we can remove this test.